airsonic-main/src/main/java/org/airsonic/player/controller/InternalHelpController.java

```diff
@@ -257,14 +258,18 @@ private void gatherScanInfo(Map<String, Object> map) {
     private void gatherIndexInfo(Map<String, Object> map) {
         SortedMap<String, IndexStatistics> indexStats = new TreeMap<>();
         for (IndexType indexType : IndexType.values()) {
-            try (IndexReader reader = indexManager.getSearcher(indexType).getIndexReader()) {
-                IndexStatistics stat = new IndexStatistics();
-                stat.setName(indexType.name());
+            IndexStatistics stat = new IndexStatistics();
+            IndexSearcher searcher = indexManager.getSearcher(indexType);
+            stat.setName(indexType.name());
+            indexStats.put(indexType.name(), stat);
+            if (searcher != null) {
+                IndexReader reader = searcher.getIndexReader();
                 stat.setCount(reader.numDocs());
                 stat.setDeletedCount(reader.numDeletedDocs());
-                indexStats.put(indexType.name(), stat);
-            } catch (IOException e) {
-                LOG.debug("Unable to gather information about {} index", indexType.name(), e);
+                indexManager.release(IndexType.ALBUM, searcher);
```

Comment:
Strictly speaking, it doesn't seem to need to be released here. 
But in the worst case it just does nothing.

Reply:
Fixed because IndexType was specified incorrectly.

IndexType.ALBUM -> indexType